examples/jsm/renderers/webgpu/WebGPURenderer.js

```diff
@@ -1,1233 +1,29 @@
-import { GPUIndexFormat, GPUTextureFormat, GPUFeatureName, GPULoadOp } from './constants.js';
-import WebGPUAnimation from './WebGPUAnimation.js';
-import WebGPURenderObjects from './WebGPURenderObjects.js';
-import WebGPUAttributes from './WebGPUAttributes.js';
-import WebGPUGeometries from './WebGPUGeometries.js';
-import WebGPUInfo from './WebGPUInfo.js';
-import WebGPUProperties from './WebGPUProperties.js';
-import WebGPURenderPipelines from './WebGPURenderPipelines.js';
-import WebGPUComputePipelines from './WebGPUComputePipelines.js';
-import WebGPUBindings from './WebGPUBindings.js';
-import WebGPURenderLists from './WebGPURenderLists.js';
-import WebGPURenderStates from './WebGPURenderStates.js';
-import WebGPUTextures from './WebGPUTextures.js';
-import WebGPUBackground from './WebGPUBackground.js';
-import WebGPUNodes from './nodes/WebGPUNodes.js';
-import WebGPUUtils from './WebGPUUtils.js';
-import { Frustum, Matrix4, Vector3, Color, SRGBColorSpace, NoToneMapping, DepthFormat } from 'three';
+import Renderer from '../common/Renderer.js';
+import WebGPUBackend from './backends/webgpu/WebGPUBackend.js';
```

Comment:
Why we need this `backend/webgpu/`, why we cannot place it in the same folder?
`renderers/webgpu/WebGPURenderer.js` should use `renderers/webgpu/WebGPUBackend.js` and `renderers/webgl/WebGLRenderer.js` should use `renderers/webgl/WebGLBackend.js`. Or, if we are okay to do a breaking change, we can remove WebGLRenderer and WebGPURenderer and tell user to do `new Renderer( new WebGLBackend() or new WebGPUBackend() )`.
**There should not be** such thing as WebGPURenderer using WebGLBackend. This will extremely confuse the users.